zerver/actions/alert_words.py

```diff
@@ -1,20 +1,28 @@
 from typing import Iterable, Sequence
 
-from zerver.lib.alert_words import add_user_alert_words, remove_user_alert_words
+from zerver.lib.alert_words import (
+    WatchedPhraseData,
+    add_user_watched_phrases,
+    remove_user_watched_phrases,
+)
 from zerver.models import UserProfile
 from zerver.tornado.django_api import send_event
 
 
-def notify_alert_words(user_profile: UserProfile, words: Sequence[str]) -> None:
-    event = dict(type="alert_words", alert_words=words)
+def notify_watched_phrases(user_profile: UserProfile, phrases: Sequence[WatchedPhraseData]) -> None:
```

Comment:
I'm thinking, should we use the term `phrase`/`phrases` only for the string, and not for `WatchedPhraseData` object. Currently, the terms aren't used consistently in the PR.